stock-portfolio-monitor/src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { createContext } from 'react';
+import './App.css';
+import { Market } from './market/Market';
+import MarketState from './market/MarketState';
+import { Portfolio } from './portfolio/Portfolio';
+import PortfolioState from './portfolio/PortfolioState';
+import Modal from "react-modal"
```

Comment:
`eslint` is not configured properly. Try to install `"airbnb-typescript"` and `"airbnb/hooks"` presets that cover React rules as well. And then configure/turn off some rules and update the codebase.